src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -297,6 +301,14 @@ export const MyVideosView = () => {
               />
             )}
             <Grid maxColumns={null} onResize={handleOnResizeGrid}>
+              {currentPage === 0 && (
+                <NewVideoTile to={absoluteRoutes.studio.editVideo()}>
+                  <TextAndIconWrapper>
+                    <StyledIcon />
+                    <StyledText variant="body2">Upload new video</StyledText>
+                  </TextAndIconWrapper>
+                </NewVideoTile>
+              )}
```

Comment:
This should likely handle loading state in some way. Currently this is clickable once other items are in loading state. For uniformity I would display some kind of placeholder for it as well